tools/data-handler/src/calculate.ts

```diff
@@ -193,38 +194,44 @@ export class Calculate {
   }
 
   // Once card specific files have been done, write the the imports
-  private async generateImports() {
-    const destinationFile = join(
-      Calculate.project.paths.calculationFolder,
-      Calculate.importFileName,
-    );
-
-    const folders = [
-      Calculate.project.paths.calculationResourcesFolder,
-      Calculate.project.paths.calculationCardsFolder,
-    ];
-
+  private async generateImports(folder: string, destinationFile: string) {
     let importsContent: string = '';
+    const files: string[] = getFilesSync(folder);
 
-    for (const folder of folders) {
-      const files: string[] = getFilesSync(folder);
-
-      // Helper to remove extension from filename.
-      function removeExtension(file: string) {
-        const index = file.lastIndexOf('.');
-        return index === -1 ? file : file.substring(0, index);
-      }
-
-      importsContent += `% ${folder}\n`;
-      for (const file of files) {
-        importsContent += `#include "${resolve(join(folder, removeExtension(file) + '.lp')).replace(/\\/g, '/')}".\n`;
-      }
-      importsContent += '\n';
+    // Helper to remove extension from filename.
+    function removeExtension(file: string) {
```

Comment:
Can't you just use `path.parse(file).name`?

Reply:
Good catch